components/integration/replace-camel/code/camel-bbe/for-integrations.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Language Built for Integrations'
+description: 'Ballerina excels with its first-class services, listeners, and clients, making integration capabilities central to its design. Build scalable integrations effortlessly, thanks to its strong type system and native support for JSON and XML. Experience the simplicity and readability of Ballerina for seamless integration development.'
+url: 'https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-zapier-samples/gdrive-new-event-to-slack-message'
```

Comment:
In other pages, the urls point to the main.bal
eg:- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/components/ai/code/ai-bbe/create-ai-powered-apis.md

Reply:
Samples should link to ballerina-guides

Reply:
We'll point to bal files. We need to decide if we are gonna have the code for competitor scenarios in ballerina-guides tho

Reply:
We'll point to bal files. We need to decide if we are gonna have the code for competitor scenarios in ballerina-guides tho